### DISCUSSION TITLE
Stellarium on iPad

### DISCUSSION THREAD

**Discussion by Trickeryphotery** [Stellarium/stellarium#1883] [Q&A]
Hi all.
I have just purchased Stellarium for iPad as my computer is too old to get the full version. I have seen the full version working on  a site called Star Safari and it looked great, so I was disappointed when I did not have the system requirements. 
The reason for this message is that having purchased Stellarium for iPad for £12.00 I find it has very little functionality. No where near as good as the full version so needless to say am very disappointed with purchase.
Can anybody relate?

**Comment by gzotti** [comment]
Stellarium for iPad is not discussed on this site. Please contact its official support channels, we know nothing more than its existence. 
System requirement for the desktop version is basically a PC not older than about ten years and yes, you can still run some older version on XP. My oldest system where Stellarium works is from 2008 (with dedicated graphics card of that era), although meanwhile on Linux.

**Comment by Trickeryphotery** [comment]
Thanks for the reply. I am a Mac user. Are there old versions for 10.10.5?

**Reply by alex-w** [reply]
see version 0.19.1

**Reply by Trickeryphotery** [reply]
Thank you
